README.md

```diff
@@ -1,10 +1,12 @@
-# CompoundKit JS SDK
+# Compound Kit SDK
 
 [![Lint](https://github.com/dinngo/compound-kit-js-sdk/actions/workflows/lint.yml/badge.svg)](https://github.com/dinngo/compound-kit-js-sdk/actions/workflows/lint.yml)
 [![Unit Test](https://github.com/dinngo/compound-kit-js-sdk/actions/workflows/unit-test.yml/badge.svg)](https://github.com/dinngo/compound-kit-js-sdk/actions/workflows/unit-test.yml)
 [![npm version](https://badge.fury.io/js/@protocolink%2Fcompound-kit.svg)](https://www.npmjs.com/package/@protocolink/compound-kit)
 
-The sdk for CompoundKit.
+Compound Kit empowers developers to rapidly build intent-centric applications and enhances the user experience for the Compound protocol. This repository includes the Compound Kit SDK.
```

Comment:
Could mention that the SDK is designed to handle Compound Kit API without directly dealing with API requests and responses, and include the link to https://github.com/dinngo/compound-kit-api.